flatliners/prometheusendpoint.py

```diff
@@ -0,0 +1,57 @@
+from time import sleep, time
+from collections import defaultdict
+from .baseflatliner import BaseFlatliner
+import dateparser
+import os
+
+from prometheus_client import Gauge, start_http_server
+
+# Set up logging
+import logging
+_LOGGER = logging.getLogger(__name__)
+
+class PrometheusEndpoint(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+        # Published Stale metrics are removed once every three times metric data is collected from prometheus
+        self.pruning_interval = os.getenv("FLT_METRIC_CHUNK_SIZE","5m")
```

Comment:
On a next PR, you should make this an option for the constructor. 
The environment variable should not leak into here...

Reply:
Or maybe do right in this PR :)

Reply:
Should be fixed now

Reply:
Should I move `        self.pruning_interval = 3 * round((dateparser.parse('now')-dateparser.parse(self.pruning_interval)).total_seconds())
` to the app.py instead?